.vitepress/config.ts

```diff
@@ -30,12 +30,16 @@ function getNav() {
 function getGuideSidebar() {
   return [
     {
-      text: 'Getting started',
+      text: 'Get started',
```

Comment:
This is a common expression. 

I agree that this is grammatically dubious, but this is what people normally expect.

Reply:
I'd advocate for a shorter form but I'm generally ok with both in this particular case